src/styles/main.scss

```diff
@@ -0,0 +1,69 @@
+$daySize: 100px;
+$countDays: 31;
+$gap: 1px;
+$calendarWidth: ($daySize * 7) + 10;
```

Comment:
The calculation for `$calendarWidth` seems incorrect. It should be `$daySize * 7 + $gap * 6` to account for the gaps between the days.